sources/packages/backend/libs/integrations/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt.processing.service.ts

```diff
@@ -62,6 +62,13 @@ export class DisbursementReceiptProcessingService {
       this.esdcConfig.ftpResponseFolder,
       fileSearch,
     );
+    // Throw an error if there are no files to be processed.
```

Comment:
Please provide an an error on why do we set the process summary to error when file is not present. 

`Business reason from what we know is, the file is expected to be present when the scheduler runs, and it should be considered as error if the file is not present for disbursement receipts.`